app/Actions/Update/Apply.php

```diff
@@ -38,10 +42,10 @@ public function __construct(
 	 */
 	private function check_prod_env_allow_migration(array &$output): bool
 	{
-		if (Config::get('app.env') == 'production') {
+		if (Config::get('app.env') === 'production') {
 			// @codeCoverageIgnoreStart
 			// we cannot code cov this part. APP_ENV is dev in testing mode.
-			if (Configs::get_value('force_migration_in_production') == '1') {
+			if (Configs::getValueAsBool('force_migration_in_production')) {
```

Comment:
```suggestion
			if (Configs::getValueAsBool('force_migration_in_production', false)) {
```

Why is there no explicit default in this case? We seem to be providing them pretty much everywhere else...